package.json

```diff
@@ -83,7 +84,7 @@
     "mkdirp": "0.5.1",
     "mocha": "2.5.3",
     "phantomjs-polyfill-find": "ptim/phantomjs-polyfill-find",
-    "phantomjs-prebuilt": "2.1.7",
+    "phantomjs-prebuilt": "^2.1.14",
```

Comment:
Inserted a `^`, which is inconsistent with the rest of the dependencies.

Now that we have lock files, perhaps best-practice has changed? (E.g. all dependencies could be brought back to [carat ranges](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004), and we can pull upgrades early in each dev cycle.)